skimage/data/__init__.py

```diff
@@ -930,3 +931,25 @@ def lfw_subset():
 
     """
     return np.load(_fetch('data/lfw_subset.npy'))
+
+
+def brain():
+    """Subset of data from the University of North Carolina Volume Rendering
+    Test Data Set.
+
+    The full dataset is available at [1]_.
+
+    Returns
+    -------
+    image : (10, 256, 256) uint16 ndarray
+
+    Notes
+    -----
+    The 3D volume consists of 10 layers from the larger volume.
+
+    References
+    ----------
+    .. [1] https://graphics.stanford.edu/data/voldata/
+
+    """
+    return _load("data/brain.tiff")
```

Comment:
I think the other functions all call _load only because _load calls _fetch internally. If we are going to change this one, should we change the rest of the functions?

Reply:
Apologies, I didn't see that `_load` calls `_fetch`, I was looking just above at `lfw_subset()`, which calls both. The whole `pooch` thing only started recently, so we are still figuring it out! 😅

Reply:
closed